toolkit/tools/internal/sliceutils/sliceutils.go

```diff
@@ -55,3 +55,12 @@ func StringsSetToSlice(inputSet map[string]bool) []string {
 
 	return outputSlice[:index]
 }
+
+// RemoveDuplicateStrings will remove duplicate entries from a string slice
+func RemoveDuplicateStrings(packList []string) (deduplicatedPackList []string) {
+	packListSet := map[string]bool{}
+	for _, entry := range packList {
+		packListSet[entry] = true
+	}
+	return StringsSetToSlice(packListSet)
+}
```

Comment:
One thing I missed with my previous suggestion is re-naming the variables to something more generic, since `sliceutils` is just a helper for all kinds of slices:
```suggestion
func RemoveDuplicateStrings(stringsSlice []string) []string {
	stringsSet := map[string]bool{}
	for _, entry := range stringsSlice {
		stringsSet[entry] = true
	}
	return StringsSetToSlice(stringsSet)
}
```